script/Arcueid/Day 8/08_00_ARC08_6.txt

```diff
@@ -666,27 +668,27 @@ I speak out the thing that I never got to say from my heart.
 [sha:a361b70c191067336cc6bb1c0fcfaec65c1641df]{
 -- Page 341, Offset 20522.
 -- ―――心底から、安心できる。
-―――Relieving from the bottom of my heart.
+―――Relieving, from the bottom of my heart.
 }
 [sha:2ce4a83ca06ec5bfeaadded8e716cf5c911031fc]{
 -- Page 342, Offset 20523.
 -- 俺の視界は壊れている。
-My vision is being destroyed.
+My vision is broken.
 }
 [sha:101d6a1eb759e52afe8d97b194194c41055c0208]{
 -- Page 342, Offset 20524.
 -- 遠野志貴の認識は<穢|けが>れている。
-My cognition is being corrupted.
+Tohno Shiki's cognition is corrupted.
 }
 [sha:23114532bc69fe87a24ac85e6ff2c84ee270a210]{
 -- Page 342, Offset 20525.
 -- <お|・><ま|・><え|・><は|・><本|・><来|・>、<こ|・><こ|・><に|・><居|・><て|・><は|・><い|・><け|・><な|・><い|・><も|・><の|・><だ|・>。
-I shouldn't even be here in the first place.
+You shouldn't even be here in the first place.
 }
 [sha:439cc0bcfe90fb844c018eb39025ead3ae211745]{
 -- Page 342, Offset 20526.
 -- ……そんな、七年前からずっと分かりきっていた結論が、こいつの前では薄れてくれる気がする。
-...That conclusion that I've known since seven years ago feels faded away when I'm in front of her.
+...These foregone conclusions that have been clear to me since seven years ago, they all seem to fade away when I'm with her.
```

Comment:
...These foregone conclusions that have been clear to me since seven years ago all seem to fade away when I'm with her.